tests/conftest.py

```diff
@@ -130,6 +131,7 @@ def test_filters_random_with_mask(sample_config: dict, tmpdir, image_random: np.
     filters.extract_channel()
     filters.extract_pixels()
     filters.images["pixels"] = image_random
+    print(f"filters.filename : {filters.filename}")
```

Comment:
Extraneous debugging line left in (by me I think, sorry).

Reply:
👍